core/src/main/scala/tech/beshu/ror/accesscontrol/domain.scala

```diff
@@ -288,6 +294,8 @@ object domain {
     }
   }
   object IndexName {
+    implicit val indexNameShow: Show[IndexName] = Show.show(_.value.value)
+    implicit val caseMappingEqualityIndexName: CaseMappingEquality[IndexName] = CaseMappingEquality.instance(identity)
```

Comment:
I doubt we need it (for all non-userid types) - Eq should be enough

Reply:
We need this for wildcard patterns